landa/organization_management/doctype/external_contact/external_contact.py

```diff
@@ -38,4 +41,4 @@ def revert_series(self):
 		number_part_len = len(self.name.split("-")[-1])
 		key = self.name[:-number_part_len] + "." + "#" * number_part_len
 
-		revert_series_if_last(key, self.name)
+		revert_series_if_last(key, self.name)
```

Comment:
```suggestion
		revert_series_if_last(key, self.name)

```
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file